testing/src/test/java/org/oppia/android/testing/math/PolynomialSubjectTest.kt

```diff
@@ -11,85 +11,92 @@ import org.oppia.android.app.model.Real
 @RunWith(JUnit4::class)
 class PolynomialSubjectTest {
 
+  // Helper function to create a polynomial term
```

Comment:
```suggestion
  /** Helper function to create a polynomial term. */
```
Here and below: KDoc is required for public functions as opposed to comments.